components/VCVerification.tsx

```diff
@@ -20,7 +20,7 @@ export const VCVerification: React.FC = ({wellknown, isVerified}: any) => {
         <React.Fragment>
           <VerifiedIcon />
           <Text
-            testID="verificationStatus"
+            testID="statusValue"
```

Comment:
let's rename it back to `verificationStatus` and we will ask QA to change this identifier in automation